### DISCUSSION TITLE
setState and tooltip redrawn

### DISCUSSION THREAD

**Discussion by Diielle** [bensonarafat/super_tooltip#104] [Q&A]
HI,

I want to make a tooltip that is made up of two parts as if they were two slides,

When I press a button (located on the tooltips) from the first slide I have to move to the second

If I call a setState in the onTap function the tooltip is not redrawn, what should I do?

Thank you

**Comment by Diielle** [comment]
sorry, the problem is that the onTap function is not called at all

**Reply by bensonarafat** [reply]
Hi @Diielle 
Are you trying to close/open the tooltip. 
You can use the controller to perform that. 
Hope this will help.
